kotlin-js/src/jsMain/kotlin/js/promise/Promise.kt

```diff
@@ -11,3 +12,9 @@ inline fun <S> Promise.Companion.all(
     values: ReadonlyArray<PromiseResult<S>>,
 ): Promise<ReadonlyArray<S>> =
     all(promise = values.unsafeCast<ReadonlyArray<Promise<S>>>())
+
+suspend fun <T> Promise<T>.asResult(): Result<T> =
```

Comment:
`Result` instance is just created here from received data and it does not proxy something from `Promise`. I propose to use `toResult` name here.

Reply:
Done